docs/current_docs/getting-started/quickstarts/blueprint/index.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "Use a Blueprint Module"
+slug: "/getting-started/quickstarts/blueprint"
+description: "Use a Blueprint Module"
+keywords: ["quickstart", "ci", "dagger", "blueprint"]
+---
+
+# Use a Blueprint Module
```

Comment:
Maybe its worth also explaining that contextual argument are directly take from the created dagger module and not the blueprint context's?

Could be done in the contextual argument page though.